### DISCUSSION TITLE
Customizing the color of TextColumn

### DISCUSSION THREAD

**Discussion by Samoylenko666** [filamentphp/filament#5501] [Help] [answered]
in the documentation I saw use

```
TextColumn::make('title')
     ->description(fn (Post $record): string => $record->description)
```
In this closure example, we have access to $record

How do you look at adding value access to the `->color()` method as well?

To be able to do something like this:

`Tables\Columns\TextColumn::make('is_something')->color(fn($value) => $value ? 'danger' : 'default' )`

**Comment by danharrin** [comment, marked as answer]
Maybe try `fn($record) => $record->is_something ? 'danger' : 'default' `
